src/components/bottom_bar/bottom_bar.ts

```diff
@@ -152,6 +152,17 @@ export class BottomBar extends Component<Props, SpreadsheetChildEnv> {
         textColor: sheet.isVisible ? undefined : "grey",
         execute: (env) => {
           env.model.dispatch("ACTIVATE_SHEET", { sheetIdFrom: from, sheetIdTo: sheetId });
+
+          // Scrolls the sheet list to ensure the specified sheet is visible.
+          const sheetEl = this.bottomBarRef.el!.querySelector<HTMLElement>(
```

Comment:
you can move this code in a useEffect of BottomBarSheet, like so
```typescript
useEffect(
      (sheetId) => {
        if (this.props.sheetId === sheetId) {
          this.scrollToSheet();
        }
      },
      () => [this.env.model.getters.getActiveSheetId()]
```
I think it's better to let the component scroll itself instead of a querySelector which assumes the DOM structure of its child component.
Of course the arguments you added to the scrollIntoView could be updated the same way in `scrollToSheet`
    );
```